bot/internal/github/github.go

```diff
@@ -561,8 +565,15 @@ func (c *Client) CreateComment(ctx context.Context, organization string, reposit
 	return nil
 }
 
-// ListComments lists all comemnts on an issue or PR.
-func (c *Client) ListComments(ctx context.Context, organization string, repository string, number int) ([]string, error) {
+// Comment represents an "issue comment" on a GitHub issue or pull request.
+// This does not include comments that are part of reviews.
+type Comment struct {
+	Author string // the GitHub username of the author
+	Body   string // the text of the comment
```

Comment:
Nit: Use godoc-style comments instead on top of the fields?

Reply:
godoc still renders it nicdely, and I kind of like when the `string` is aligned.

```
➜ go doc "github.com/gravitational/shared-workflows/bot/internal/github.Comment"
package github // import "github.com/gravitational/shared-workflows/bot/internal/github"

type Comment struct {
	Author string // the GitHub username of the author
	Body   string // the text of the comment
}
    Comment represents an "issue comment" on a GitHub issue or pull request.
    This does not include comments that are part of reviews.


shared-workflows/bot on zmb3/get-flake-excludes
➜ go doc "github.com/gravitational/shared-workflows/bot/internal/github.Comment.Author"
package github // import "github.com/gravitational/shared-workflows/bot/internal/github"

type Comment struct {
    Author string  // the GitHub username of the author

    // ... other fields elided ...
}
```